### ISSUE TITLE
Ignore *.pth files by default in AutoConfig

### ISSUE THREAD

**Issue by gitlawr** [modelscope/modelscope#1004]
**Checklist**

* I have searched the tutorial on modelscope  [doc-site](https://modelscope.cn/docs)
* I have searched related issues but cannot get the expected help.
* The bug has not been fixed in the latest version.

**Describe the bug**
When using AutoConfig.from_pretrained(), it downloads the weight file.

**To Reproduce**
Run
```
from modelscope import AutoConfig

AutoConfig.from_pretrained("LLM-Research/Llama-3.2-11B-Vision-Instruct")
```

**Result**

It downloads the 21.26GB `original/consolidated.pth` file.

We can override `ignore_file_pattern` for that. Does it make more sense to ignore *.pth files as the default?

**Other Info**
In comparison, `transformers` only downloads the config.json file.
```
from transformers import AutoConfig

AutoConfig.from_pretrained(
    "meta-llama/Llama-3.2-11B-Vision-Instruct",
    token="xxx",
)
```

Model hub related: @liuyhwangyh

**Comment by yingdachen** [comment]
we will look into this.

**Comment by yingdachen** [comment]
fixed in the latest code pr #1008
